app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,24 @@
+# show each individual question
+get '/questions/:id' do
+  question = Question.find(params[:id] + 1)
+  survey = TakenSurvey.find(params[:survey_id])
+  if defined?(question)
+    erb :'/questions/show', locals: {question: question, survey: survey}
+  else
+    erb :'/stats'
+  end
+end
+
+# post to the answers table to track a users response for a specific question then redirect to the next question, else return to the same page and
+post '/questions' do
+  answer = Answer.new(params)
+  if answer.save
+    redirect '/questions/<%= params[:question.id] %> + 1'
```

Comment:
you are incrementing here (so incrementing twice) incrementing should be in the model if you are incrementing, as a method  :dash:  :lips: